toolchain/sem_ir/type_iterator.h

```diff
@@ -187,14 +187,16 @@ class TypeIterator::Step {
   struct End {};
   // Iteration is complete.
   struct Done {};
+  // Iteration found an error.
+  struct Error {};
```

Comment:
Is this used? Shouldn't type iterator be returning it somewhere?

Reply:
Fixed